src/shared_modules/content_manager/tests/unit/executionContext_test.hpp

```diff
@@ -49,6 +52,16 @@ class ExecutionContextTest : public ::testing::Test
             }
         )"_json;
     }
+
+    /**
+     * @brief Tear down routine for each test fixture.
+     *
+     */
+    void TearDown() override
+    {
+        std::filesystem::remove_all("/tmp/content_manager");
```

Comment:
Nitpick: Define this as a const. Like m_databasePath

Reply:
Done!